deploy-firebase-example/cloudbuild.yaml

```diff
@@ -0,0 +1,4 @@
+steps: 
+- name: gcr.io/$PROJECT_ID/firebase
+  entrypoint: bash
```

Comment:
This can be simplified to use the default entrypoint (though it also works as-is).

Reply:
Adjarabet. Com